ludwig/features/image_feature.py

```diff
@@ -68,6 +69,7 @@ class ImageFeatureMixin:
         'height': {'type': 'integer', 'minimum': 0},
         'width': {'type': 'integer', 'minimum': 0},
         'num_channels': {'type': 'integer', 'minimum': 0},
+        'infer_image_num_channels': {'type': 'boolean'},
```

Comment:
I'm trying to think if there's a scenario where the user wants to infer the height and width but not the channels. Right now I'm leaning towards `infer_image_dimensions` just handling all 3 to simplify things. Do you have some thoughts on this?